pkg/storer/sample.go

```diff
@@ -149,18 +160,22 @@ func (db *DB) ReserveSample(
 				chunk, err := db.ChunkStore().Get(ctx, chItem.Address)
 				if err != nil {
 					wstat.ChunkLoadFailed++
+					db.metrics.ReserveSampleChunksLoadFailed.Inc()
 					db.logger.Debug("failed loading chunk", "chunk_address", chItem.Address, "error", err)
 					continue
 				}
 
 				wstat.ChunkLoadDuration += time.Since(chunkLoadStart)
+				db.metrics.ReserveSampleChunksLoaded.Inc()
```

Comment:
this is TotalIterated - (BelowBalanceIgnored + RogueChunk + ChunkLoadFailed). There are metrics for all of them.